guides/user/managed-delivery/getting-started/index.md

```diff
@@ -159,7 +160,7 @@ Make sure your git repository is accessible by Spinnaker.
 
 Since Spinnaker is the source of truth (and not git) you need to submit your managed resource configuration to Spinnaker every time there are changes.
```

Comment:
```suggestion
Since Spinnaker is the source of truth (and not Git) you need to submit your managed resource configuration to Spinnaker every time there are changes.
```